pkg/liqonet/iptables/iptables.go

```diff
@@ -710,13 +753,26 @@ func checkTep(tep *netv1alpha1.TunnelEndpoint) error {
 			Reason:    errors.ValidCIDR,
 		}
 	}
+	if err := utils.IsValidCIDR(tep.Status.LocalExternalCIDR); err != nil {
+		return &errors.WrongParameter{
+			Parameter: localExternalCIDR,
+			Reason:    errors.ValidCIDR,
+		}
+	}
 	if err := utils.IsValidCIDR(tep.Status.LocalNATPodCIDR); tep.Status.LocalNATPodCIDR != consts.DefaultCIDRValue &&
 		err != nil {
 		return &errors.WrongParameter{
 			Parameter: localNATPodCIDR,
 			Reason:    errors.ValidCIDR,
 		}
 	}
+	if err := utils.IsValidCIDR(tep.Status.LocalNATExternalCIDR); tep.Status.LocalNATExternalCIDR != consts.DefaultCIDRValue &&
```

Comment:
Since this method has a lot of checks, I would add comment on it by explaining what is the overall purpose. I guess the objective is to detect collisions and malformed CIDRs but a comment would definitely help.